cmd/tailscale/cli/web.html

```diff
@@ -71,7 +71,7 @@ <h5>{{.IP}}</h5>
 	{{ else }}
 	<div class="mb-6">
 		<h3 class="text-3xl font-semibold mb-3">Log in</h3>
-		<p class="text-gray-700">Get started by logging in to your Tailscale network. Or,&nbsp;learn&nbsp;more at <a
+		<p class="text-gray-700">To get started, login to your Tailscale network. Or,&nbsp;learn&nbsp;more at <a
```

Comment:
Changed

Reply:
You changed it the wrong way. It was correct before. The verb has a space in it. "login" is a noun.